packages/components/src/components/Signin/Signin.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Signin } from '.';
+
+// builds EIP-4361 compliant message.
+function buildMessage(
+  domain: string,
+  address: string,
+  uri: string,
+  version: number,
+  chainId: number,
+  nonce: number,
+  issuedAt: string,
+  statement?: string,
+  requestId?: string,
+  exp?: Date,
+  notBefore?: Date,
+  resources?: string[]
+) {
+  let message = domain + ' wants you to sign in with your Ethereum account: \n' + address + '\n\n';
```

Comment:
Could we use a string literal template her instead ?

Reply:
ahh yes we could. I did a lot of nonsense in school and to get better at practical coding lol. I appreciate the advice!